modules/sildisco/src/IdPDisco.php

```diff
@@ -68,6 +69,8 @@ private function getSPEntityIDAndReducedIdpList(): array
 
     /**
      * @inheritDoc
+     * @throws MetadataNotFound|ConfigurationError
+     * @throws \Exception
```

Comment:
Is it a different Exception? Why the slash?

Reply:
I think it's because of this: `yii\db\Exception`